lib/btree2/kdtree.c

```diff
@@ -524,7 +524,7 @@ int kdtree_knn(struct kdtree *t, double *c, int *uid, double *d, int k, int *ski
     if (skip)
 	sn.uid = *skip;
 
-    maxdist = 1.0 / 0.0;
+    sscanf("inf", "%lf", &maxdist);
```

Comment:
In #2681, the code was changed to use `INFINITY`, but there's a note that it might not for MSVC (at that time)